mapsforge-samples-android/src/main/java/org/mapsforge/samples/android/OverlayMapViewer.java

```diff
@@ -190,6 +193,52 @@ public boolean onTap(LatLong geoPoint, Point viewPosition,
         layers.add(circle);
         layers.add(marker1);
         layers.add(tappableCircle);
+        layers.add(createPolygonWithHoles(anchorPolygonWithHoles));
+    }
+
+    private Polygon createPolygonWithHoles(final LatLong anchor) {
+        Paint paintFill = Utils.createPaint(
+                AndroidGraphicFactory.INSTANCE.createColor(Color.GREEN), 2,
+                Style.FILL);
+        Paint paintStroke = Utils.createPaint(
+                AndroidGraphicFactory.INSTANCE.createColor(Color.RED), 10,
+                Style.STROKE);
+        Polygon polygonWithHoles = new Polygon(paintFill, paintStroke, AndroidGraphicFactory.INSTANCE) {
+            @Override
+            public boolean onTap(LatLong tapLatLong, Point layerXY, Point tapXY) {
+                if (contains(tapLatLong)) {
+                    Toast.makeText(OverlayMapViewer.this, "PolygonWithHoles tap\n" + tapLatLong, Toast.LENGTH_SHORT).show();
+                    return true;
+                }
+                return false;
+            }
+        };
+        polygonWithHoles.addPoint(anchor);
+        polygonWithHoles.addPoint(addFraction(anchor, 10, 10));
+        polygonWithHoles.addPoint(addFraction(anchor, 50, 10));
+        polygonWithHoles.addPoint(addFraction(anchor, 50, 80));
+        polygonWithHoles.addPoint(addFraction(anchor, 10, 80));
+
+        polygonWithHoles.addHole(Arrays.asList(
+                addFraction(anchor, 20, 20),
+                addFraction(anchor, 25, 30),
+                addFraction(anchor, 15, 30)));
+        polygonWithHoles.addHole(Arrays.asList(
+                addFraction(anchor, 40, 40),
+                addFraction(anchor, 45, 40),
+                addFraction(anchor, 45, 60),
+                addFraction(anchor, 40, 60)));
+        polygonWithHoles.addHole(Arrays.asList(
+                addFraction(anchor, 20, 40),
+                addFraction(anchor, 35, 70),
+                addFraction(anchor, 25, 60),
+                addFraction(anchor, 15, 60)));
+
+        return polygonWithHoles;
+    }
+
+    private static LatLong addFraction(final LatLong latLon, final int latAdd, final int lonAdd) {
+        return new LatLong(latLon.getLatitude() + ((double) latAdd / 5000d) , latLon.getLongitude() + ((double) lonAdd / 5000d) );
```

Comment:
Can you please check the formatting here?

Reply:
Same here, I was not sure what exactly you referred to. I inserted a line break and removed the blank before the comma and the last parenthesis. If you meant something else I can further modify code

Reply:
> removed the blank before the comma and the last parenthesis

:+1: 

> I inserted a line break

The new line needs more indentation (not very important).

Reply:
done